packages/perspective-viewer-d3fc/src/js/axis/sunburstLabel.js

```diff
@@ -0,0 +1,32 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2017, the Perspective Authors.
+ *
+ * This file is part of the Perspective library, distributed under the terms of
+ * the Apache License 2.0.  The full license can be found in the LICENSE file.
+ *
+ */
+
+import {select} from "d3";
+
+export const labelVisible = d => d.y1 <= 3 && d.y0 >= 1 && (d.y1 - d.y0) * (d.x1 - d.x0) > 0.06;
+
+export function labelTransform(d, radius) {
```

Comment:
These labelling helpers seem to relate more to the sunburst series rather than the axis.